src/config/mod.rs

```diff
@@ -884,6 +900,8 @@ impl Default for WriteCfConfig {
             format_version: 2,
             checksum: ChecksumType::CRC32c,
             max_compactions: 0,
+            ttl: None,
```

Comment:
@tabokie Should we simply disable ttl and periodic compaction by default?

Reply:
I don't have strong opinion on this. The large compaction triggered by ttl will eventually be triggered by other reasons. The only difference is that ttl is more deterministic.

Reply:
> I don't have strong opinion on this. The large compaction triggered by ttl will eventually be triggered by other reasons. The only difference is that ttl is more deterministic.

Is there any restriction when pick the input files for cross level compaction.  For example, by common sense it's not reasonable to compact a 4MB L5 file with 600GB L6 files, regardless of the reason.  I''m wondering if there's such restriction or can we add it?

Reply:
No. The restriction is dangerous. E.g. a SST containing range tombstone would overlap with lots of files, adding this restriction will make it impossible to free up space.